engine/battle/init_battle_variables.asm

```diff
@@ -20,12 +20,16 @@ InitBattleVariables:
 	ld [hli], a ; wPlayerHPBarColor
 	ld [hl], a ; wEnemyHPBarColor
 	ld hl, wCanEvolveFlags
-	ld b, $3c
-.loop
+	ld b, $3c ; What is this?
+.loop ; What does this do exactly?
 	ld [hli], a
 	dec b
 	jr nz, .loop
-	inc a ; POUND
+	; Force a specific move in TestBattle.
+	; The game implicitly runs "ld a, NO_MOVE"
+	; when the instruction is absent.
+	; Increment a to load POUND instead.
+	inc a
```

Comment:
No "when the instruction is absent"; the instruction is *not* absent.
```suggestion
	; Force a specific move in TestBattle.
	; The game implicitly runs "ld a, NO_MOVE"
	; when the instruction is absent.
	; Increment a to load POUND instead.
	inc a
```
```suggestion
	; During TestBattle, a is 0 here, so this sets a to move 1, POUND.
	inc a
```

Reply:
Actually, what about when it's not during `TestBattle`? Is this code accessible then?

Reply:
> the instruction is _not_ absent

I was trying to explain it in less technical terms, but true.

> Actually, what about when it's not during `TestBattle`?

If it's a regular battle, then you are free to use whatever move is available to you.

Reply:
Okay then, how's this:
```
; Force player to use POUND (move id 1) during TestBattle.
```

Reply:
Looks good! 😄